operations/host_spawn.go

```diff
@@ -480,6 +480,58 @@ func hostStart() cli.Command {
 	}
 }
 
+func hostSSH() cli.Command {
+	const (
+		keyFlagName = "key"
```

Comment:
I don't understand what you mean by Evergreen conventions. Do we usually use `-i ` for something else?

Reply:
Ah what I mean is that in many host routes, we use "key" or "k" to reference the ssh key, with the message "name or value of an public key to use".  For ssh, "-i" is "Selects a file from which the identity (private key) for public key authentication is read." We could instead say something like "path to the private key (reference -i for ssh)

Reply:
I just realized typing that out that -k is for the public key, so it's probably okay to change this to -i.